src/main/java/com/mars/review/infrastructure/ReviewJpaRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.mars.review.infrastructure;
+
+import com.mars.review.domain.Review;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import java.util.List;
+
+@Repository
+public interface ReviewJpaRepository extends JpaRepository<Review, Long> {
```

Comment:
fyi; `@Repository` 붙여도 상관없는데 JpaRepository interface는 안 붙여도 될 거예여
```suggestion
public interface ReviewJpaRepository extends JpaRepository<Review, Long> {
```

Reply:
넵넵